addons/sale/views/sale_views.xml

```diff
@@ -320,7 +320,12 @@
                                 <label for="date_order" string="Order Date"/>
                             </div>
                             <field name="date_order" attrs="{'required': [('state', 'in', ['sale', 'done'])], 'invisible': [('state', 'in', ['draft', 'sent'])]}" nolabel="1"/>
-                            <field name="pricelist_id" groups="product.group_product_pricelist" options="{'no_open':True,'no_create': True}"/>
+                            <field name="is_change_pricelist" invisible="1"/>
+                            <label for="pricelist_id" groups="product.group_product_pricelist"/>
+                            <div groups="product.group_product_pricelist" class="o_row">
+                                <field name="pricelist_id" options="{'no_open':True,'no_create': True}"/>
+                                <button name="update_pricelist" type="object" string=" Update Prices" class="btn-link mb-1 px-0" icon="fa-refresh" confirm="Are you sure to recompute all the prices based on the new pricelist?" attrs="{'invisible': ['|', ('is_change_pricelist', '=', False), ('state', 'in', ['sale', 'done','cancel'])]}"/>
```

Comment:
Please split the line to avoid having too long lines

```xml
<button name="" type="" string=""
    class="" icon=""
    confirm=""
    attrs=""/>

```

Reply:
@Feyensv Done.